cmd/contour/servecontext.go

```diff
@@ -153,6 +153,10 @@ type serveContext struct {
 	//
 	// If this field not specified, all supported versions are accepted.
 	DefaultHTTPVersions []string `yaml:"default-http-versions"`
+
+	// ClusterConfig holds various configurable Envoy cluster values that can
+	// be set in the config file.
+	ClusterConfig `yaml:"cluster,omitempty"`
```

Comment:
I don't really have a better suggestion for naming here. I guess this could just be a top-level field in the config file too. 🤷‍♂️

Reply:
Yup that's where I started, but if we ever needed to add another "cluster" type config it's nice to have this section for it.